### ISSUE TITLE
simulated data h2o model validation

### ISSUE THREAD

**Issue by Devashish13** [xinghuq/DeepGenomeScan#3]
Hi, 

I have implemented the h2o model on the simulated data provided in the package. 

I have performed 5 fold cross validation and getting MSE around 1.25 and Rsquare 0.25, still I am getting the outlier SNP associated with QT1,QT2 with high True Positive  rate (TPR).

could you explain the intuition behind this?

Thanks,

**Comment by xinghuq** [comment]
Hi,  @Devashish13 , I do not get what you mean. Have you implemented the analysis of 100 replicates (simulated datasets)?
1.   "still I am getting the outlier SNP associated with QT1,QT2 with high True Positive rate (TPR)." Is this not what you are expecting? 
2.  MSE depends on the magnitude of the data. which model has R square of 0.25? there are three QTs, for model with responses variable env3 (discrete environment variable), the R square is definitely small.
Hope this helps.
If you still have questions, post your codes will be much helpful for me to understand what you question means.

Cheers,

XQ

**Comment by Devashish13** [comment]
Hi @xinghuq , Thanks for the response.

I have implemented single run with file sim1.csv provided in the package. 

By 100 replicates, do you mean I have to implement the same code 100 times or there are 100 different simulation files.

In the package there is only one file sim1.csv, and also the simulated data is not available at the link provided in the paper.
I am attaching the link below
https://datadryad.org/review?doi=doi:10.5061/dryad.1s7v5

By the above question I wanted to understand that 

1) is there a strong relationship between Rsquare and feature importance of the variables? 
2) can we identify the variables contributing in the predictive power of the neural network even if Rsquare is not that high(say Rsquare = 0.4 on validation set)?

Thanks

**Comment by xinghuq** [comment]
Hi @Devashish13 ,

What is your purpose of running the analysis of simulations? If you have your own datasets, just run the analysis on your own data, the simulations in the example is a single run test with tuning limited hyperparameters. Your should follow the description of the analysis in the paper as well as the examples so that work on your own data. However, the datasets you send doesn't seem exist.

" The page you are looking for doesn't exist.

Please use the navigation menu above to find the page you're looking for."

1. is there a strong relationship between Rsquare and feature importance of the variables?
More important variables contributing to predicting the response variables will have high R-square.  But we can not ensure the certain relationship between Rsquare and feature importance. 

3. can we identify the variables contributing in the predictive power of the neural network even if Rsquare is not that high(say Rsquare = 0.4 on validation set)?

You can still identify the variables even if the Rsqure is smaller, but that means the features contributing less important to the model.

**Comment by Devashish13** [comment]
thanks @xinghuq , my issues are resolved.
